images/router/haproxy/conf/haproxy-config.template

```diff
@@ -152,8 +152,10 @@ defaults
   # backend service or alternatively, you can send a custom 503 or 404 error.
   #
   # server openshift_backend 127.0.0.1:8080
+  errorfile 504 {{ env "ROUTER_ERRORFILE_504" "/var/lib/haproxy/conf/error-page-504.http" }}
   errorfile 503 {{ env "ROUTER_ERRORFILE_503" "/var/lib/haproxy/conf/error-page-503.http" }}
   errorfile 404 {{ env "ROUTER_ERRORFILE_404" "/var/lib/haproxy/conf/error-page-404.http" }}
+  errorfile 403 {{ env "ROUTER_ERRORFILE_403" "/var/lib/haproxy/conf/error-page-403.http" }}
```

Comment:
Also even though the lines are not currently ordered, we would ask you to order these lines by error code number: 403 404 503 504.  Note that these capitalized env variables have exist so that people can customize their error pages if they don't want the default.